thirdparty/kyaml/runfn/runfn.go

```diff
@@ -152,38 +131,21 @@ func (r RunFns) getNodesAndFilters() (
 		return nil, nil, outputPkg, err
 	}
 
-	fltrs, err := r.getFilters(buff.Nodes)
+	fltrs, err := r.getFilters()
 	if err != nil {
 		return nil, nil, outputPkg, err
 	}
 	return buff, fltrs, outputPkg, nil
 }
 
-func (r RunFns) getFilters(nodes []*yaml.RNode) ([]kio.Filter, error) {
-	var fltrs []kio.Filter
-
-	// fns from annotations on the input resources
-	f, err := r.getFunctionsFromInput(nodes)
-	if err != nil {
-		return nil, err
-	}
-	fltrs = append(fltrs, f...)
-
-	// fns from directories specified on the struct
-	f, err = r.getFunctionsFromFunctionPaths()
-	if err != nil {
-		return nil, err
-	}
-	fltrs = append(fltrs, f...)
-
+func (r RunFns) getFilters() ([]kio.Filter, error) {
```

Comment:
Read the code some more. I think we can call r.getFunctionFilters() directly at the caller of this function and get rid of these multiple wrappers to just get the functionFilters.

Reply:
removed the function wrappers.